### PR TITLE
blog: add blog post on the affect of containerization on databases

### PR DESCRIPTION
**Pull request by shenying1023**
This is a blog talking about containerization and its influence on databases.

### CODE REVIEW

**Comment by sftim** [comment]
/hold
pending assignment of publication date

File: content/en/blog/_posts/2024-03-16-Does-Containerization-Affect-The-Performance-Of-Databases/Usage-of-containerized-workloads-by-category.png
Comment (sftim) [inline content/en/blog/_posts/2024-03-16-Does-Containerization-Affect-The-Performance-Of-Databases/Usage-of-containerized-workloads-by-category.png:1]: Hi. I imagine we'd need a copyright license agreement from Datadog; can you provide that?

**Review by sftim — COMMENTED** [review]
This looks very similar to https://kubeblocks.io/blog/Does-containerization-affect-the-performance-of-databases

Can you explain the similarity? We can only accept original articles.

File: content/en/blog/_posts/2024-03-16-Does-Containerization-Affect-The-Performance-Of-Databases/Usage-of-containerized-workloads-by-category.png
Comment (shenying1023) [reply content/en/blog/_posts/2024-03-16-Does-Containerization-Affect-The-Performance-Of-Databases/Usage-of-containerized-workloads-by-category.png:1]: Hi, thanks for your reply. The link you pasted is the official website of our company. Sorry for the negligence and maybe next time I should upload the original articles here. I may just close this pr. Thanks again for your attention.

**Pull request closed without merging**